innerprod/fullysec/damgard_test.go

```diff
@@ -26,14 +26,24 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestFullySec_DamgardDDH(t *testing.T) {
-	l := 3
-	bound := big.NewInt(1000)
-	sampler := sample.NewUniformRange(new(big.Int).Neg(bound), bound)
-	modulusLength := 64
-
-	damgard, err := fullysec.NewDamgard(l, modulusLength, bound)
+type damgardTestParam struct {
+	name string
+	modulusLength int
+	precomputed bool
+}
 
+func testFullySec_DamgardDDHFromParam(t *testing.T, param damgardTestParam) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; func testFullySec_DamgardDDHFromParam should be testFullySecDamgardDDHFromParam](https://app.codacy.com/app/tilenmarc/gofe_2/pullRequest?prid=4432579)